### ISSUE TITLE
Adding Cluster controls to the Dashboard

### ISSUE THREAD

**Issue by jakirkham** [dask/distributed#1931]
Would be nice if the Cluster controls (currently provided as IPython Widget object), also were provided on a tab in the Dashboard. Options to add or remove workers even when running in Adaptive mode would be nice. That way if we know there is an operation that we need to speed up ATM, we can go in and do a manual override temporarily.

**Comment by martindurant** [comment]
Yes! I was imagining how one might do this over a year ago, but didn't get very far - that was before the widgets and the standardisation of the adaptive process.

**Comment by jakirkham** [comment]
Are you referring to PR ( https://github.com/dask/distributed/pull/862 )? Had looked at that and thought it was interesting as well. Do you think that approach is still reasonable or should we try something new? What stumbling blocks did you encounter when working on that PR?

**Comment by mrocklin** [comment]
No objection from me.  Some concerns that will need to be addressed though:

1.  This likely needs to be configurable based on the Cluster object
2.  This likely needs to be off by default for security concerns.  We make no attempt to secure the bokeh server
3.  Note that the current widget is made with ipywidgets and not with bokeh, so you shouldn't expect current code to carry over

**Comment by jakirkham** [comment]
Presumably it would get that functionality for free as it would display the dashboard.

**Comment by rabernat** [comment]
I think such a jupyterlab extension would be really useful. In pangeo.pydata.org, each user creates their own cluster on a per-notebook basis. But it really makes more sense to do it on a per-session basis. Having a jupyerlab extension to manage the cluster would be a perfect solution.
